1-JavaScript/Week4/challenges/2-sum-three-entries.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Credit to https://adventofcode.com/ for this exercise
+In the list below you have an array of numbers. The goal is to find the three numbers that add up to 2020.
+Once you have found those numbers, multiply the numbers and store the result of that in the result variable.
+ */
+
+
+const list = [1721, 979, 366, 299, 675, 1456];
+let result;
+    
+// Write your code here
+const a = list.forEach((elementA) => {
+    let d = 2020 - elementA
+    const b = list.forEach((elementB) => {
+     let c = d - elementB 
+        for (let x = 0; x< list.length; x++) {
+           if (c === list[x]) {
+                return   result = elementA * elementB * c;
+            }
+        }        
+    });  
+});
```

Comment:
congrats! this exercise shows the expected result. However, I would like to see if you can get the same result but, like the previous exercise, stop looping as soon as you find the result.

Hint: It is not possible to stop looping using forEach method, so you should follow a similar approach as you did in the previous exercise